### ISSUE TITLE
define multiple separators

### ISSUE THREAD

**Issue by galantra** [glfharris/echelon#9]
feature suggestion: define multiple separators which work simultaneously
e. g. "::", "_"

Esp. useful for shared decks which don't follow the same naming rules

**Comment by glfharris** [comment]
Yeah it's a useful idea, thanks!

I need to read up on the config updating anki does before I can do something like this, as there's a danger of breaking it.

**Comment by JayDugger** [comment]
I'd sure like this feature. Multiple separators, specifically "::" and "_", would clear up the sidebar clutter in my collection.

**Comment by glfharris** [comment]
Yeah I can definitely see how it would be useful, but comes with the risk of cluttering up the filter bar more than it is already or slowing down the browser.
With the filter string, you might have to have every permutation of tag separator (e.g. "tag:Something::* or tag:Something_*) , or query before hand to see what tags exist.

I'll have a go at testing it at some point.
